### DISCUSSION TITLE
To be a VTuber !!!

### DISCUSSION THREAD

**Discussion by Charmve** [YunYouJun/YunYouJun#5] [Ideas]
Hi, YunYouJun

So great your VTuber making job is ! I am also a VTuber-liker. This ia my [OpenVTuber-虚拟アイドル共享计划](https://github.com/DeepVTuber). Do you have interest on making a VTuber, a new VTuber? We can have a cooperation. 

If you wanna go, please let me know.

@Charmve

**Comment by YunYouJun** [comment]
Yeah, that's interesting. Maybe we can give it a try.
But I have an important exam in the last few days, so I can't communicate with you further right away.

**Comment by Charmve** [comment]
It's glad that I received your letter. No worries. We can keep a touch with Wechat that I will text to you in email.

Good luck for your exam!
